File: code/backend/src/main/java/com/bycoders/cnab/dominio/services/TipoTransacaoService.java
package com.bycoders.cnab.dominio.services;

import javax.enterprise.context.ApplicationScoped;
import javax.inject.Inject;

import com.bycoders.cnab.dominio.repositories.TipoTransacaoRepository;

@ApplicationScoped
public class TipoTransacaoService {
    
    @Inject
    TipoTransacaoRepository tipoTransacaoRepository;
    
    public void popularBancoTipoTransacao() {
        System.out.println("Chegou aqui no tipoTransacaoService");
        // final String insert_sql = "INSERT INTO public.tipotransacao (tipo, descricao, natureza, sinal ) VALUES(?,?, ?, ?)";
        // tipoTransacaoRepository.getEntityManager().createNativeQuery(insert_sql).setParameter(1, 9).setParameter(2, "Aluguel").setParameter(3, "Saída").setParameter(4, "-").executeUpdate();
    }
}

File: backEnd/src/Services/tipoTransacaoService.js
import TipoTransacaoRepository from '../repo/tipoTransacaoRepo';

class TipoTransacaoService
{
	async getAll()
	{
		return await TipoTransacaoRepository.getAll();
	}

	async addNew(cnab)
	{
		return await TipoTransacaoRepository.addNew(cnab)
	}
}
export default new TipoTransacaoService();